raiden-ts/docs-source/SDK-Development.md

```diff
@@ -193,22 +194,22 @@ Reducers are one of the simplest parts to write. Just some heads-up:
 - Try to always be as type strict as possible. lodash's `get` and `set` usually aren't typesafe (because of their nested sweetness), so add explicit declarations and narrowing checks whenver needed, to be warned when changing anything that could break a reducer.
 - Don't overthink reducers: you can always assume you're receiving a valid/consistent state and action, so no need to add too much checks besides the typing ones, just be sure to always return a valid/consistent state as well and we should be good.
 - Each `action` type must be handled only once, in a single and very specific place, to avoid issues with the `reduce-reducers` pattern used in the root reducer declaration.
-- The state must be JSON-serializable. Try to stay on the primitive types as much as possible, unless there's a good reason to have an instance there, and it must be serializable (with custom `io-ts` codecs, like `BigNumber`s).
+- The state must be JSON-serializable. Try to stay on the primitive types as much as possible, unless there's a good reason to have an instance there, and it must be serializable (with custom `io-ts` codecs, like `BigNumber`).
```

Comment:
The new action creators logic helps with that: they include a `codec` property, which contains an `io-ts` codec comprising of the whole action, and can be used to [de]serialize actions. This was made and is used currently only for the `ConfirmableActions` subset.